desktop/apps/editorial_features/components/venice_2017/client/index.coffee

```diff
@@ -45,6 +47,12 @@ module.exports = class VeniceView extends Backbone.View
     @VeniceVideoView.trigger 'swapVideo',
```

Comment:
Ah thanks, i just updated stagingwriter's curation so it should work now. The section was published but didn't have a video file which caused the video view to be sad. I will put in some messaging on the writer side!